app/src/editor/canvas/index.jsx

```diff
@@ -375,7 +375,15 @@ const CanvasLoader = withRouter(withErrorBoundary(ErrorComponentView)(class Canv
         this.unmounted = true
     }
 
-    init() {
+    async init() {
+        if (!this.props.match.params.id) {
+            // if no id, create new
+            const newCanvas = await services.create()
+            if (this.unmounted) { return }
+            this.props.history.replace(`${links.editor.canvasEditor}/${newCanvas.id}`)
```

Comment:
I'd use the $routes helper here for generating the url

Reply:
good idea, we should start using this more.

Reply:
I think I'll do this in another pass and fix it for the whole canvas/dashboard editor.